cfgov/jinja2/v1/_includes/organisms/carousel.html

```diff
@@ -27,7 +27,7 @@
 <section class="o-carousel_item">
     <div class="o-carousel_item-wrapper">
         <div class="o-carousel_item-text">
-            <h2>{{ item.title }}</h2>
+            <h3 class="h2">{{ item.title }}</h3>
```

Comment:
It feels hacky using classes like this to restyle headings but it feels better than adding CSS rules. LMK if there's a better approach.

Reply:
I think this is the hacky approach we use elsewhere, so :+1: